### ISSUE TITLE
Question about scope

### ISSUE THREAD

**Issue by wlandau** [RConsortium/r-advisory-database#14]
I am wondering about the specific types of issues acceptable for this database. Is the scope restricted to only security issues such as malware and exploitable code, or may I contribute advisories for other matters that would violate the [R-multiverse Acceptable Use Policy](https://r-multiverse.org/aup.html) (such as copyright violation)?

**Comment by glin** [comment]
Yes, ultimately this database is meant to feed into the Open Source Vulnerabilities database, so I'd say the scope is limited to only security vulnerabilities.

**Comment by wlandau** [comment]
Thanks, that makes sense.
